SafeFoodie_Fresh/test/list_history_test.dart

```diff
@@ -0,0 +1,138 @@
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility in the flutter_test package. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+//import 'dart:html';
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+//import 'package:safefoodie_fresh/screens/list.dart';
+import 'package:safefoodie_fresh/main.dart';
+import 'package:safefoodie_fresh/screens/homepage/list_history.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:settings_ui/settings_ui.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+void main() {
+  /*testWidgets('Counter increments smoke test', (WidgetTester tester) async {

```

Comment:
Please remove this uneeded unit test. Thank you!

Reply:
Will do!